advanced_guides/stop_words.md

```diff
@@ -84,7 +84,7 @@ Lets make a search on this dataset
 
 `The Road to Hong Kong` is the first film in our search results. It is there because of the recurrence of the words "the" and "of" in the title and description. However, it is the oldest film in the documents offered.
 
-In order to allow the search engine to focus more on the release date, let's add stop-words that prevent giving so much importance to documents that have no specificity other than the recurrence of stop-words.
+In order to allow the search engine to focus more on the release date, let's add stop-words that prevent giving so much importance to documents that have no specificity other than the recurrence of very common words.
```

Comment:
I know you didn't add this line but it's too long (maybe 2 sentences would be better). And I don't understand really well this part "that prevent giving so much importance to documents that have no specificity other than the recurrence of very common words".
I mean, I understand what does a stop word because I know the subject, but when I read the sentence, it's too "heavy".

Reply:
A new proposition has been made.